src/pyggdrasil/distances/_interface.py

```diff
@@ -7,6 +7,37 @@
 _IntegerTreeRoot: TypeAlias = TreeNode[int, Any]
 
 
+class TreeSimilarity(Protocol):
```

Comment:
Currently, we only use `TreeSimilarity` correct? -`TreeDistance` is here only for future tree dissimilarity measures to be implemented?

Reply:
Yes – I adjusted `TreeDistance` protocol, so that all these methods have `calculate: (tree1, tree2) -> float` method. We will be able to use them interchangeably when drawing the plots.

(Later we may see that some of them are more suitable for diagnosing convergence issues, but right now it's too early to say which one to go with...)